MoreLinq.Test/ZipLongestTest.cs

```diff
@@ -31,13 +33,13 @@ public class ZipLongestTest
         public static readonly IEnumerable<ITestCaseData> TestData =
             from e in new[]
             {
-                new { A = Seq<int>(  ), B = Seq("foo", "bar", "baz"), Result = Seq((0, "foo"), (0, "bar"), (0, "baz")) },
-                new { A = Seq(1      ), B = Seq("foo", "bar", "baz"), Result = Seq((1, "foo"), (0, "bar"), (0, "baz")) },
-                new { A = Seq(1, 2   ), B = Seq("foo", "bar", "baz"), Result = Seq((1, "foo"), (2, "bar"), (0, "baz")) },
-                new { A = Seq(1, 2, 3), B = Seq<string>(           ), Result = Seq((1, null ), (2, null ), (3, (string) null)) },
-                new { A = Seq(1, 2, 3), B = Seq("foo"              ), Result = Seq((1, "foo"), (2, null ), (3, null )) },
-                new { A = Seq(1, 2, 3), B = Seq("foo", "bar"       ), Result = Seq((1, "foo"), (2, "bar"), (3, null )) },
-                new { A = Seq(1, 2, 3), B = Seq("foo", "bar", "baz"), Result = Seq((1, "foo"), (2, "bar"), (3, "baz")) },
+                new { A = Seq<int>(  ), B = Seq("foo", "bar", "baz"), Result = Seq<(int, string?)>((0, "foo"), (0, "bar"), (0, "baz")) },
```

Comment:
Similar comments to #882. My preferred way of handling it, but can be changed with overt preference.